src/interfaces.ts

```diff
@@ -149,16 +158,23 @@ export enum State {
   Connecting = TrackPlayer.STATE_CONNECTING,
 }
 
-export interface TrackMetadata {
-  duration?: number
-  title: string
-  artist: string
+interface TrackMetadataBase {
+  title?: string
   album?: string
+  artist?: string
+  duration?: number
+  artwork?: string | ResourceObject
+}
+
+export interface TrackMetadata extends TrackMetadataBase {
   description?: string
   genre?: string
   date?: string
   rating?: number | boolean
-  artwork?: string | ResourceObject
+}
```

Comment:
Why do we have to split out `description`, `genre`, `date` and `rating` from the now playing metadata?